kubetest2-gke/deployer/build/gke_make.go

```diff
@@ -35,6 +36,8 @@ const (
 	pack         gkeBuildAction = "package"
 	stage        gkeBuildAction = "push-gcs"
 	printVersion gkeBuildAction = "print-version"
+
+	gkeMinorVersionRegex string = "^v(\\d\\.\\d+).*$"
```

Comment:
nit: I think we typically keep regex as vars with MustCompile to avoid issue in the regex itself

Reply:
Also I like `(?<Year>\d{4})-(?<Month>\d{2})-(?<Day>\d{2})` using the named capture groups. Really helps understand what is what. and you don't need to just index the match. you can say match["minor_version"]

Reply:
> nit: I think we typically keep regex as vars with MustCompile to avoid issue in the regex itself
Done.
> named capture group
I think the named group is not needed in this case since the variable name has implied what it is.